automation/jleechanorg_pr_automation/openai_automation/codex_github_mentions.py

```
@@ -331,33 +411,96 @@ async def find_github_mention_tasks(self) -> List[Dict[str, str]]:
+                    continue
+                seen.add(href)
+                deduped.append(task)
+
+            if self.task_limit is not None:
+                deduped = deduped[: self.task_limit]
+
+            print(f"✅ Prepared {len(deduped)} task link(s) for processing")
+            logger.info(f"Prepared {len(deduped)} task link(s) across tabs")
+            return deduped
 
         except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)